token-swap/program/src/instruction.rs

```diff
@@ -107,7 +131,8 @@ pub enum SwapInstruction {
     ///   Must be empty, not owned by swap authority
     ///   6. `[writable]` Pool Token Account to deposit the initial pool token
     ///   supply.  Must be empty, not owned by swap authority.
-    ///   7. '[]` Token program id
+    ///   7. '[]` Freeze authority. Optional.
```

Comment:
This isn't actually passed as an account, no need to have it as a comment

Reply:
Oops!